doc/whats-new.rst

```diff
@@ -50,7 +50,7 @@ New Features
 - Implement :py:func:`median` and :py:func:`nanmedian` for dask arrays. This works by rechunking
```

Comment:
we can't reference those since there is no documentation page generated. Alternatively, you could reference `Dataset.median` and `DataArray.median`
```suggestion
 - Implement ``median`` and ``nanmedian`` for dask arrays. This works by rechunking
```